tests/config/functional/ini/pylintrc_with_quoted_init_hook.ini

```diff
@@ -0,0 +1,4 @@
+# Check that we support quoted init-hooks
+# See https://github.com/PyCQA/pylint/issues/7006
+[MASTER]
```

Comment:
```suggestion
[MAIN]
```

Reply:
Good to see both are still supported 😄

Reply:
> both are still supported

Yeah, the resulting shit-storm would have soaked us to the bone otherwise.